app/models/concerns/declarable.rb

```diff
@@ -2,36 +2,49 @@ module Declarable
   extend ActiveSupport::Concern
   include Formatter
 
+  COMMON_UNION_MEASURE_ORDER = [
+    Sequel.desc(:measures__measure_generating_regulation_id),
+    Sequel.desc(:measures__measure_generating_regulation_role),
```

Comment:
Now that we're doing a distinct on the role, postgres told me I needed to add the corresponding ordering for the role